value-fixture/src/org/immutables/fixture/custann/CustomImmutables.java

```diff
@@ -15,12 +15,16 @@
  */
 package org.immutables.fixture.custann;
 
+import org.immutables.value.Value;
+
 public class CustomImmutables {
 
   @CustomWithStyle
+  @Value.Immutable
   interface Od {}
   
   @CustomWithStyle2
+  @Value.Immutable
```

Comment:
I don't know how these were generated before. They have no immutable annotation present anywhere.

Reply:
It's specifically the test for custom annotation, `@CustomWithStyle` should be the one which replaces both `@Value.Immutable` and the associated style. So something is not working in how resources are loaded/discovered in  java > 8

Reply:
Ok for now we can leave with test basically a noop, can you just add FIXME note to either fix or discontinue the feature

Reply:
probably affected by annotationProcessorPath change